frontend/src/app/auth.service.ts

```diff
@@ -9,6 +9,7 @@ import { Observable } from 'rxjs/Observable';
 import { environment } from '../environments/environment';
 import { User } from './user.d';
 import { SuccessOrFailureAction } from './models';
+import { error } from 'selenium-webdriver';
```

Comment:
what's that? do we need really to import `selenium-webdriver` inside application?